### PR TITLE
Changes and improvements for the Block Grid Editor documentation

### PR DESCRIPTION
**Pull request by nielslyngsoe**
A few additions and changes to the Block Grid Editor documentation.

TODO:
[] Would be nice with a screenshot/gif for the new part about Scaling

### CODE REVIEW

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/build-custom-view-for-blocks.md [hunk 1]
```diff
@@ -39,28 +39,39 @@ Example:
...
-## Add the Areas Container for Block Grid Editor
+## Add the Areas for Block Grid Editor
 
-Blocks of the Block Grid Editor can have Areas. These Blocks requires their Custom View to define where the Area Container should be inserted.
+Blocks of the Block Grid Editor can have Areas. These Blocks requires their Custom View to define where the Areas should be inserted.
 
 This enables you to wrap and place the Areas as you need.
 
-The Areas is inserted through a slot, this slot will become the Area Container.
+The Areas is inserted through [Web Component Slots](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots), A slot is a native HTML tag.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/build-custom-view-for-blocks.md:48]:
```suggestion
The Areas is inserted through [Web Component Slots](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots). A slot is a native HTML tag.
```

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/build-custom-view-for-blocks.md [hunk 2]
```diff
@@ -39,28 +39,39 @@ Example:
...
+Example of a Custom View wrapping the Areas within a green border:
 
 ```html
 <div style="border:green 1px solid; padding: 20px;">
-    <slot name="area-container" part="area-container"></slot>
+    <umb-block-grid-render-area-slots></umb-block-grid-render-area-slots>
 </div>
 ```
 
+You can also specifically define where each Area should be inserted, this is relevant if you like the Areas to appear differently than the provided grid setup of Areas.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/build-custom-view-for-blocks.md:64]:
```suggestion
You can also define where each Area should be inserted. This is relevant if you like the Areas to appear differently than the provided grid setup of Areas.
```

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md [hunk 3]
```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
 
 ### Area configuration
 
+An area 
+
 ![Block Grid - Area Configuration](images/BlockGridEditor_AreasConfiguration.png)
 
-* **Alias** - The alias is printed by `GetBlockGridHTML()`. Use the alias to target the Element representing the area.
+* **Alias** - The alias is used to identify this Area.
+It begin printed by `GetBlockGridHTML()` And used as name for the Area slot in Custom Views.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md:169]: I'm not sure I understand this sentence. Could it be rewritten different to be more clear?

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md [hunk 4]
```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
...
+The alias is also available for CSS Selectors to target the HTML-Element representing a Area.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md:170]: This is a list, right? Should each new line here be a new list item, or is the text meant to be part of the "Alias" list item? 🤔

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md [hunk 5]
```diff
@@ -160,15 +161,22 @@ To scale an Area, click and drag the scale-button in the bottom-right corner of
...
 
 * **Create Button Label** - Overwrites the Create Button Label of the Area.
 
 * **Number of blocks** - Determines the total number of Blocks in an Area.
 
-* **Allowed block types** - Defines the types of Blocks or Groups of Blocks that are allowed in the Area. Optionally, you can also set how many Blocks of each type/group should be present. The Blocks can be limited for each specific Block Type in an Area.
+* **Allowed block types** - When this is empty, all Blocks with Permissions for creation in Areas, will be available. This can be overwritten by specifying the allowed Blocks.
+Define the types of Blocks or Groups of Blocks that are allowed. Additionally, you can also set how many Blocks of each type/group should be present.
```
Comment (sofietoft) [inline Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md:177]: Should this line be part of the "Allowed block types" list item in the line above?

**Review by sofietoft — COMMENTED** [review]
I've made a few comments and suggestions

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md (hunk 3 above)
Comment (nielslyngsoe) [reply Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md:169]: The addition/change is that now the Alias is not only printed to be used for CSS selectors. But is also used for the slots of which the Area is injected into a Block View.
This is important if you not want to use the Directive: `<umb-block-grid-render-area-slots></umb-block-grid-render-area-slots>` but ant to write the slots on your own.  hen you need to understand that the name of the slots are the Alias. :-D

I hope that helps, though it is quite technical, and only relevant to few.

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md (hunk 4 above)
Comment (nielslyngsoe) [reply Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md:170]: Oh,. yeah, its a list. so this was additional tekst about the alias.

**Comment by sofietoft** [comment]
Made some small adjustments to the list for the "Area configuration".
Will double-check the formatting once it's merged.

I'll merge it in now, and revisit once I've had a chance to test it tomorrow 💪

File: Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md (hunk 5 above)
Comment (nielslyngsoe) [reply Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/Block-Grid-Editor/index.md]: Yep, thats right:-)

**Pull request merged**